tests/modules/diamond/blastx/test.yml

```diff
@@ -1,18 +1,21 @@
 - name: diamond blastx test_diamond_blastx
-  command: nextflow run tests/modules/diamond/blastx -entry test_diamond_blastx -c tests/config/nextflow.config
+  command: nextflow run ./tests/modules/diamond/blastx -entry test_diamond_blastx -c ./tests/config/nextflow.config  -c ./tests/modules/diamond/blastx/nextflow.config
   tags:
-    - diamond
     - diamond/blastx
+    - diamond
   files:
+    - path: output/diamond/proteome.fasta.dmnd
+    - path: output/diamond/test.diamond_blastx.diamond.log
```

Comment:
```suggestion
    - path: output/diamond/test.diamond_blastx.log
```

Reply:
Thanks a lot for the suggestions and explaining the context!